src/lib/apiTypesV2/Subscription.cpp

```diff
@@ -0,0 +1,104 @@
+/*
+*
+* Copyright 2015 Telefonica Investigacion y Desarrollo, S.A.U
+*
+* This file is part of Orion Context Broker.
+*
+* Orion Context Broker is free software: you can redistribute it and/or
+* modify it under the terms of the GNU Affero General Public License as
+* published by the Free Software Foundation, either version 3 of the
+* License, or (at your option) any later version.
+*
+* Orion Context Broker is distributed in the hope that it will be useful,
+* but WITHOUT ANY WARRANTY; without even the implied warranty of
+* MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+* General Public License for more details.
+*
+* You should have received a copy of the GNU Affero General Public License
+* along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+*
+* For those usages not covered by this license please contact with
+* iot_support at tid dot es
+*
+* Author: Orion dev team
+*/
+
+#include "Subscription.h"
+
+#include <string>
+#include <sstream>
+#include <vector>
+
+#include "common/JsonHelper.h"
+
+using std::string;
+using std::vector;
```

Comment:
As long as all the code uses the same style, I will be satisfied. Uniformity is important.
If we later decide to use 'using' instead of explicitely showing std:: every time, then let's change the *entire* code at once. I will vote against such a change, but I have only *one* vote ...